src/autorest-core/lib/configuration.ts

```diff
@@ -393,17 +394,18 @@ export class ConfigurationView {
           let blameTree: BlameTree | null = null;
 
           try {
+            const originalPath = JSON.stringify(s.Position.path);
             while (blameTree === null) {
               try {
                 blameTree = this.DataStore.Blame(s.document, s.Position);
               } catch (e) {
                 const path = s.Position.path as string[];
                 if (path) {
-                  this.Message({
-                    Channel: Channel.Verbose,
-                    Text: `Could not find the exact path ${JSON.stringify(path)} for ${JSON.stringify(m.Text)}`
-                  });
                   if (path.length === 0) {
+                    this.Message({
```

Comment:
why? this message is there solely to raise awareness about a buggy extension, i.e. one that reports the wrong path - reporting only paths that are SO broken that you walk up to `$` seems somewhat arbitrary, muting bugs and making them go unnoticed (this was once even a warning, which was our only leverage for the tooling people to fix their rules) :P

Reply:
A couple of points:

- I watched a lot of these messages go by, and a lot of them can be caused because there is a drop in fidelity of what the path can be because of the modeler. The leaf (or leaf+1) ends up being correct, but the rest is fine. I'm not so much worried about these; the correction is fixing it up fine.

- for messages that have no useful path, then I'm willing to yell, but I don't want to see a whole bunch of messages for the same one (which happens on every iteration of the loop)

- sadly, It's clear that they don't care about these anyway. The best we can do is attach the message to the less accurate ancestor node and move along.

Reply:
> caused because there is a drop in fidelity of what the path can be because of the modeler

how is the modeler involved? 😲 

> sadly, It's clear that they don't care about these anyway. The best we can do is attach the message to the less accurate ancestor node and move along.

 well that's how we forced them to fix the rules last time and it worked great :P but okay, I guess this is causing noise since VScode subscribes to verbose messages too? Just wondered, since *verbose* messages seem to be precisely for this kind of thing, but now we're trying to not be verbose on verbose? :D